battle/BattleScene.gd

```diff
@@ -193,3 +199,23 @@ func enemies_in_range(custom_range):
 				in_range.append(enemy)
 				break
 	return in_range
+
+"""
+Ability
+"""
+
```

Comment:
Is it possible to call the _on_hand_play_card function while playing the game or will that be implemented later?

Reply:
The _on_hand_play_card function is called when the Hand node senses that one of its cards was played.  It works for me, at least.